docs/recipes/installTips.md

```diff
@@ -364,6 +363,254 @@ GUI will open!
+| NVIDIA-SMI 510.108.03   Driver Version: 510.108.03   CUDA Version: 11.6     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  NVIDIA GeForce ...  Off  | 00000000:01:00.0 Off |                  N/A |
+|  0%   49C    P8    17W / 350W |      5MiB / 24576MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+                                                                               
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|    0   N/A  N/A      1275      G   /usr/lib/xorg/Xorg                  4MiB |
++-----------------------------------------------------------------------------+
+
+```
+
+Sweet!
+
+
+### Next, Anaconda!
+
+Click here to get the ubuntu/linux package: https://www.anaconda.com/products/individual#linux
+
+this downloads a file, save it somewhere. (~/Downloads is OK)
+
+then enter the folder:
+
+`cd Downloads`
+
+and run:
+
+`bash Anaconda3-2021.05-Linux-x86_64.sh`
+
+and you get:
+```python
+Welcome to Anaconda3 2021.05
+
+In order to continue the installation process, please review the license
+agreement.
+Please, press ENTER to continue
+>>>
+```
+
+Follow prompts, so many prompts!
+
+### Next, DeepLabCut!
+
+Given this is a totally fresh install, here are a few things that also needed: 
+
+`sudo apt install libcanberra-gtk-module libcanberra-gtk3-module`
+
+DeepLabCut can be installed via:
+* Docker: We strongly recommend for Ubuntu users to use Docker (https://hub.docker.com/r/deeplabcut/deeplabcut) - it's a much more reproducible environment.
+* A conda file provided via the DeepLabCut website. This may be a bit more advanced in the development
+* A simple installation through pip: either full GUI (recommeneded) or without GUI (called: DLCLite)
+
+#### For pip instillation (GUI): 
+
+Create an enviroment: 
+
+`conda create --name DLCenvGUI python=3.8`
+
+Activate the enviroment: 
+
+`conda activate DLCenvGUI`
+
+Install DLC: 
+
+`pip install 'deeplabcut[gui,tf]'`
+
+Start the GUI with: 
+
+`python -m deeplabcut`
+
+**[Do DeepLabCut stuff.](https://deeplabcut.github.io/DeepLabCut/docs/standardDeepLabCut_UserGuide.html)**
```

Comment:
I believe this would be nicer in a bit more concise way (otherwise it spreads out a lot when reading the documentation):

```bash
conda create --name DLCenvGUI python=3.8  # Create an environment
conda activate DLCenvGUI                  # Activate it
pip install "deeplabcut[gui, tf]"         # install DLC
python -m deeplabcut                      # Start the GUI
```